ui/src/pages/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import logo from "./sml-logo-vertical-rgb-trans.png";
+import Image from "react-bootstrap/Image";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import { Link } from "react-router-dom";
+
+const Welcome: React.FC = () => (
+  <>
```

Comment:
unnecessary wrapping in fragment

Reply:
it is necessary - there are two `Container` wrapped